src/services/codefixes/fixImportNonExportedMember.ts

```diff
@@ -0,0 +1,229 @@
+                s1.propertyName || s1.name,
+                s2.propertyName || s2.name
+            )
+        );
+    }
+
+    const isVariableDeclarationListWith1Element = (
+        node: Node
+    ): node is VariableDeclarationList =>
+        !(isVariableDeclarationList(node) && node.declarations.length !== 1);
+
+    function doChange(
+        changes: textChanges.ChangeTracker,
+        sourceFile: SourceFile,
+        node: Identifier
+    ): void {
+        const moduleSymbol = sourceFile.localSymbol ?? sourceFile.symbol;
+
+        const localSymbol = moduleSymbol.valueDeclaration?.locals?.get(
+            node.escapedText
+        );
+
+        if (localSymbol === undefined) {
+            return;
+        }
+
+        // Node we need to export is a function
+        if (isFunctionSymbol(localSymbol)) {
+            const node = localSymbol.valueDeclaration;
+
+            if (node === undefined) {
+                return;
+            }
+
+            if (!isDeclarationStatement(node) && !isVariableStatement(node)) {
+                return;
+            }
+
+            return changes.insertExportModifier(sourceFile, node);
+        }
+        // Node we need to export is a variable declaration
+        else if (
+            localSymbol.valueDeclaration &&
+            isVariableDeclarationListWith1Element(
+                localSymbol.valueDeclaration.parent
+            )
+        ) {
+            const node = localSymbol.valueDeclaration.parent;
+
+            return changes.insertExportModifier(sourceFile, node);
+        }
+
+        // In all other cases => the node is a variable, and should be exported via `export {a}`
+        const namedExportDeclaration = getNamedExportDeclaration(sourceFile);
+
+        // If there is an existing export
+        if (
+            namedExportDeclaration?.exportClause &&
+            isNamedExports(namedExportDeclaration.exportClause)
+        ) {
+            return changes.replaceNode(
+                sourceFile,
+                namedExportDeclaration,
+                factory.updateExportDeclaration(
+                    /*node*/ namedExportDeclaration,
+                    /*modifiers*/ undefined,
+                    /*isTypeOnly*/ false,
+                    /*exportClause*/ factory.updateNamedExports(
+                        /*node*/ namedExportDeclaration.exportClause,
+                        /*elements*/ sortSpecifiers(
+                            namedExportDeclaration.exportClause.elements.concat(
+                                factory.createExportSpecifier(
+                                    /*isTypeOnly*/ false,
+                                    /*propertyName*/ undefined,
+                                    node
+                                )
+                            )
+                        )
+                    ),
+                    /*moduleSpecifier*/ undefined,
+                    /*assertClause*/ undefined
+                )

```

Comment:
```suggestion
                factory.updateExportDeclaration(
                    namedExportDeclaration,
                    /*modifiers*/ undefined,
                    /*isTypeOnly*/ false,
                    factory.updateNamedExports(
                        namedExportDeclaration.exportClause,
                        sortSpecifiers(
                            namedExportDeclaration.exportClause.elements.concat(
                                factory.createExportSpecifier(
                                    /*isTypeOnly*/ false,
                                    /*propertyName*/ undefined,
                                    node
                                )
                            )
                        )
                    ),
                    /*moduleSpecifier*/ undefined,
                    /*assertClause*/ undefined
                )

```

I think some of these arguments are self explanatory.

Reply:
I'm happy to remove any of these you'd like, but tbh as someone new to the project => they're not :P 

Honestly half the pain is having so many positional arguments at all!